frontend/src/app/modules/attachments/attachments.component.ts

```diff
@@ -26,23 +26,21 @@
 // See docs/COPYRIGHT.rdoc for more details.
 //++
 
-import {Component, Input, OnInit, OnDestroy} from '@angular/core';
+import {Component, ElementRef, Input, OnInit} from '@angular/core';
 import {HalResource} from 'core-app/modules/hal/resources/hal-resource';
-import {DynamicBootstrapper} from 'core-app/globals/dynamic-bootstrapper';
-import {ElementRef} from '@angular/core';
 import {HalResourceService} from 'core-app/modules/hal/services/hal-resource.service';
 import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {States} from 'core-components/states.service';
-import {componentDestroyed} from 'ng2-rx-componentdestroyed';
-import {filter, takeUntil} from 'rxjs/operators';
+import {filter} from 'rxjs/operators';
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
 
 export const attachmentsSelector = 'attachments';
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><del><span data-href="https://github.com/opf/openproject/blob/85e9c35a9c9361e0038637188a645cba25065d78/frontend/src/app/modules/attachments/attachments.component.ts#L38"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</del></td>
    </tr>
  </tbody>
</table>